src/views/pages/wallets/WalletDetailsPage/WalletDetailsPage.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="wallet-detail-outer-container" v-show="currentWallet">
+  <div v-show="currentWallet" class="wallet-detail-outer-container">
```

Comment:
for some reason, this line is swapping back and forward?  is this eslint fix?

Reply:
I don't know why it is like that, it appears in David's PRs as well

Reply:
just checked, it is a linting change from eslint
since v-show should be before class, I'll let it